swifttool/manager.py

```diff
@@ -0,0 +1,216 @@
+import copy
+import logging
+import math
+import os
+import shutil
+import stat
+import subprocess
+import sys
+import tempfile
+import time
+import yaml
+from logging.handlers import SysLogHandler
+from utils import get_devices, RING_TYPES, env
+from swift.common.ring import RingBuilder
+
+
+class CapacityManager(object):
+
+    def __init__(self, iterations=10,
+                 def_file='/etc/swift/ring_definition.yml',
+                 cluster_changes=None, cluster_config=None):
+        self.iterations = iterations
+        env.key_filename = '/home/swiftops/.ssh/id_rsa'
+        env.user = 'swiftops'
+        if cluster_changes is None or cluster_config is None:
+            self.cluster_changes, self.cluster_config = \
+                calculate_cluster_changes(def_file, iterations)
+        else:
+            self.cluster_changes = cluster_changes
+            self.cluster_config = cluster_config
+        self.logger = _setup_rsyslog()
```

Comment:
We do not use rsyslog for our logging. Giftwrap has good examples of how we have standardized on logging.